### DISCUSSION TITLE
What's the best way to find, join projects and contribute on GitHub?

### DISCUSSION THREAD

**Discussion by fudztown** [community/community#22873] [New to GitHub]
<p>Hello GitHub Community,</p>
<p>I’ve been on a user GitHub and other repos for years, but never really explored the Community. So I guess I’m largely a GitHub ‘Newbie’ in a lot of ways. I’m interested in getting involved in some new projects. I’ve got lots of skills in data and coding and also I’m a Program Manager by trade. What’s the best way to see where my skills could be put to work and help out on some projects? I also don’t know how to “contribute” effectively. Any advice/links to reading would be appreciated.</p>

**Comment by angela-crist** [comment, marked as answer]
<p>Hi <a class="mention" href="/fudztown">@fudztown</a>,</p>
<p>Thanks for being part of the GitHub Community Forum! We’re glad you’re here.</p>
<p>A great way to jump into new projects is to search for projects that have the <code>help wanted</code> and <code>good first issue</code> labels. </p>
<p>Here are a few other things you may want to check out:<br>
<a href="https://guides.github.com/" rel="nofollow noopener">GitHub Guides </a></p>
<p><a href="https://opensource.guide/how-to-contribute/" rel="nofollow noopener">How to Contribute to Open Source</a></p>
<p><a href="https://github.com/trending" rel="nofollow noopener">Trending Repositories</a></p>
<p>Also, this post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our community organized so users can more easily find information.</p>
<p>As you’ll notice, your topic is now here in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>
